git-refresher/viewing-branches-github.qmd

```diff
@@ -0,0 +1,18 @@
+---
+title: "Switching branches on GitHub"
+---
+
+To view and switch between the different branches available in a GitHub repository, follow these steps:
+
+1.  Navigate to the repository on GitHub. In this case, [Git-academy-sandbox](https://github.com/dfe-analytical-services/git-academy-sandbox).
```

Comment:
```suggestion
1.  Navigate to the repository on GitHub. In this case, [Git-academy-sandbox](https://github.com/dfe-analytical-services/git-academy-sandbox){target="_blank"}.
```

Reply:
Adding that should make it open in a new tab so that users don't lose sight of the guide when the click on the link.